runtime-common/cli11/autobuild/defines

```diff
@@ -0,0 +1,12 @@
+PKGNAME=cli11
+PKGSEC=libs
+PKGDES="CLI11 is a header-only command-line argument parser for C++11 and later."
```

Comment:
No period point or any punctuation mark at the end.

"A header-only command-line argument parser for C++11 and later"

Reply:
modified.